### DISCUSSION TITLE
practice-exam/practice-exam-7

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#282] [Comments]
# practice-exam/practice-exam-7

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-7.html

**Comment by dalibor-florian** [comment]
36. Which of the following is NOT a benefit of using AWS Lambda?
A. AWS Lambda runs code without provisioning or managing servers.
B. AWS Lambda provides resizable compute capacity in the cloud.
C. There is no charge when your AWS Lambda code is not running.
D. AWS Lambda can be called directly from any mobile app.

Shouldn't be the right answer for this question B instead of D?

**Reply by asifMuzammil** [reply]
I think you are right
